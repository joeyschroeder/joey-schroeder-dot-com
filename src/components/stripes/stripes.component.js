import React from 'react';
import { Stripe } from '../stripe/stripe.component';
import styles from './stripes.style.scss';

export function Stripes() {
  return (
    <div className={styles.root}>
      <Stripe />
      <Stripe />
      <Stripe />
      <Stripe />
      <Stripe />
    </div>
  );
}
