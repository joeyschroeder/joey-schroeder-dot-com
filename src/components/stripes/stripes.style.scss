@import "../../styles/config/variables";

.root {
  left: 0;
  position: absolute;
  top: 0;
  width: 100%;
}
