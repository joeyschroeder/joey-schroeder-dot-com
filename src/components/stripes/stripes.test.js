import React from 'react';
import { create } from 'react-test-renderer';
import { Stripes } from './stripes.component';

describe('<Stripes />', () => {
  it('should render correctly', () => {
    const component = create(<Stripes />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
