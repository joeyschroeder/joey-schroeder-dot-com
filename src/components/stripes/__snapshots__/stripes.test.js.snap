// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stripes /> should render correctly 1`] = `
<div
  className="root"
>
  <div
    className="root"
  />
  <div
    className="root"
  />
  <div
    className="root"
  />
  <div
    className="root"
  />
  <div
    className="root"
  />
</div>
`;
